packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -66,17 +65,23 @@ contract DependencyRegistryV0 is
     function _onlySupportedCoreContract(
         address coreContractAddress
     ) internal view {
+        DependencyRegistryStorageLib.Storage
+            storage ds = DependencyRegistryStorageLib.s();
+
         require(
-            _supportedCoreContracts.contains(coreContractAddress),
+            ds.supportedCoreContracts.contains(coreContractAddress),
             "Core contract not supported"
         );
     }
 
     function _onlyExistingDependency(
         bytes32 dependencyNameAndVersion
     ) internal view {
+        DependencyRegistryStorageLib.Storage
+            storage ds = DependencyRegistryStorageLib.s();
+
         require(
-            _dependencyNameVersionIds.contains(dependencyNameAndVersion),
+            ds.dependencyNameVersionIds.contains(dependencyNameAndVersion),
             "Dependency does not exist"
```

Comment:
In cases where you are only grabbing a single variable from the Storage struct, you could do the following. However, I think how you have done it probably results in a bit more readable code, and is good as-is. Curious what your opinion is.
```
// can accomplish the check without defining an intermediate storage pointer `ds`
DependencyRegistryStorageLib.s().supportedCoreContracts.contains(
                coreContractAddress
            )
```

Reply:
Yeah, I don't think the intermediate storage pointer makes things meaningfully more readable, updated!